Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml

```diff
@@ -92,18 +95,20 @@
     sprite: Objects/Weapons/Melee/claymore.rsi
     state: icon
   - type: MeleeWeapon
-    attackRate: 0.5
+    attackRate: 0.65
+    range: 1.85
     damage:
       types:
-        Slash: 24
+        Slash: 19
   - type: MeleeBloodletter
     bloodReduction:
       types:
         Slash: 40.0
   - type: MeleeStaminaCost
-    swing: 20
+    swing: 10
```

Comment:
I like splitting the stamina cost for this.

Reply:
the claymore already deals 19 damage and the range has buffed to 1.85, the swing was halved so it you consume more stamina when you also hit your opponent